src/ziggurat/tracer.clj

```diff
@@ -0,0 +1,33 @@
+(ns ziggurat.tracer
```

Comment:
Docstring!!

Please add a substantial docstring which briefly explains the tracing flow in ziggurat - initialization of tracer objet, adding spans, how and where all the tracer object should be passed around, etc, etc.

The docstring should also briefly explain the format of the span this code is using, or a link to external doc can be provided.

Reply:
Awesome, Thanks!

Reply:
added

Reply:
Thanks.